aten/src/ATen/NestedTensorImpl.cpp

```diff
@@ -85,5 +85,19 @@ const char* NestedTensorImpl::tensorimpl_type_name() const {
   return "NestedTensorImpl";
 }
 
+const std::vector<int64_t> & NestedTensorImpl::get_ibd_offsets() {
+  if (ibd_offsets_.empty()) {
+    this->initialize_unbinding_info_();
+  }
```

Comment:
Is it possible to initialize the unbinding info when a nested tensor is created (i.e. in the factory function) instead of doing it lazily?